frontend_build/src/read_bundle_plugins.js

```diff
@@ -93,7 +93,7 @@ var readBundlePlugins = function(base_dir) {
       return bundle.coreAPISpec;
     }).length > 1
   ) {
-    throw new RangeError('You have more than one coreAPISpec modification specified.');
+    console.warn('You have more than one coreAPISpec modification specified.');
```

Comment:
`logging.info` instead?

Reply:
Oh yeah, clever me - I forgot I implemented a logging module in the node js side.